src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Displays user profile avatar image with/withour user name
+ */
+const Avatar = ({ className, size, userName, imgUrl }) => {
```

Comment:
add `userEmail` or `gravatarEmail` prop and if true use override `imgUrl` to 
```js
import md5 from 'md5';
import { trim } from 'lodash';

imgUrl = `https://www.gravatar.com/avatar/${md5(trim(email.toLowerCase()))}?s=${size}`
```
This way we have the flexibility of adding a custom avatar image using `imgUrl` or using gravatar if we can